<?xml version="1.0"?>
<launch>
	<arg name="controller_freq" default="15" />
	<arg name="occupancy_grid" default="/map" />

	<param name="youbot_robot_description"
			command="$(find xacro)/xacro.py '$(find robot_control)/hardware/youbot_base.urdf.xacro'" />

	<node pkg="robot_control" type="fake_driver" name="fake_driver" output="screen">
		<param name="vel_topic" value="cmd_vel" />
		<param name="controller_freq" value="$(arg controller_freq)" />
		<param name="occupancy_grid" value="$(arg occupancy_grid)" />
	</node>

	<node pkg="robot_control" type="robot_trainer" name="robot_trainer" output="screen">

	</node>

	<node pkg="robot_state_publisher" type="robot_state_publisher" name="youbot_state"  output="screen">
      <remap from="robot_description" to="youbot_robot_description" />
      <remap from="joint_states" to="youbot_joint_states" />
    </node>

</launch>

